README.md

```diff
@@ -18,8 +18,18 @@ Add spree_ams to your Spree store's Gemfile:
 gem 'spree_ams', :github => 'hhff/spree_ams', :branch => '2-3-alpha'
 ```
 
+Make sure `config.action_controller.asset_host` is set in your environment config if not running in production.
```

Comment:
right now the code is setup use that unless you are in production.

so it will break in other environments too

https://github.com/hhff/spree_ams/blob/master/app/serializers/spree/image_serializer.rb#L31

what do?